lib/config/configuration.js

```diff
@@ -453,7 +453,7 @@ Configuration.prototype._processConfig = function(config) {
     }
 
     if (options.hasOwnProperty('maxErrors')) {
-        this._loadMaxError(options.maxErrors);
+        this._loadMaxError(options.maxErrors === 'null' ? null : options.maxErrors);
```

Comment:
Hm, i wonder who this usually done for such flags, like passing "0", can't remember any popular commands with similar logic.

Anyway, could you add tests for it?

/cc @zxqfox